app/pubspec.lock

```diff
@@ -12,7 +12,7 @@ packages:
       name: analyzer
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.30.0+2"
+    version: "0.30.0+4"
```

Comment:
Can these version upgrades go in a separate change?

Reply:
They can go before but not after, because my change alone causes the following:

```
$ pub run test 
... Skipped all tests
```

Only an upgrade of `package:test` seems to make it run tests again.